internal/pkg/sourcetype/testdata/src/crosspkg/test.go

```diff
@@ -18,8 +18,9 @@ import "sourcetype"
 
 type AliasStruct = sourcetype.Source // want AliasStruct:"source type"
 
+type NamedType sourcetype.Source // want NamedType:"source type"
```

Comment:
This is a side effect of using Facts in the fieldtags analyzer. Because `NamedType`'s underlying type has a tagged field, it is identified as a source. This was not the case previously, because knowledge that `sourcetype.Source` has a tagged field was not being propagated to dependencies.